_projects/25-tasty-scala-2.md

```diff
@@ -0,0 +1,19 @@
+---
+label: tastyScala2
+name: TASTy Reader For Scala 2
+web:
+github:
+origin:
+contributors: [jamie]
+status: Active
+type: project
+active: true
+home: false
+description: Read Scala 3 libraries from Scala 2
```

Comment:
Consider: *Consume* Scala 3 libraries from Scala 2. We can *read* TASTy, or *consume* libraries, but *reading* libraries is confusing, IMO.

Reply:
I agree Consume sounds better